fabric-crash-report-info-v1/src/main/java/net/legacyfabric/fabric/mixin/crashreport/CrashReportMixin.java

```diff
@@ -32,13 +34,14 @@
 import net.fabricmc.loader.api.ModContainer;
 
 @Mixin(CrashReport.class)
-public abstract class MixinCrashReport {
+public class CrashReportMixin {
```

Comment:
Why did you remove the `abstract` ?  Its a good practice to make your mixins `abstract` because it means you can easily add abstract members (eg. shadow methods) without the compiler complaining.
it also means you can't accidentally do `new CrashReportMixin` anywhere else in your code by mistake.